packages/next/build/webpack-config.ts

```diff
@@ -1113,6 +1128,26 @@ export default async function getBaseWebpackConfig(
               },
             ]
           : []),
+        ...(isFlight
+          ? [
+              {
+                test: /\.client\.(js|ts)x?$/,
+                use: {
+                  loader: 'next-react-flight-loader',
+                },
+              },
+            ]
+          : []),
+        ...(!isServer
+          ? [
+              {
+                test: /\.server\.(js|ts)x?$/,
+                use: {
+                  loader: 'next-react-flight-exclude-server-loader',
+                },
+              },
+            ]
+          : []),
```

Comment:
For Flight pages, I was expecting that you would modify `next/build/entries.ts` to add a `flight` one:

```tsx
const client: webpack5.EntryObject = {}
const server: webpack5.EntryObject = {}
const flight: webpack5.EntryObject = {}
```

Ignoring the `serverless` target. For each page you would create a `flight` entry using your loader (similar to how `next-serverless-loader` is used there). Based on whether or not the `absolutePagePath` was `*.server.*` or not, you would either import and use it directly or generate a stub (an empty stub is fine for now, but eventually I hope all pages are implemented in terms of RSC).

Reply:
Likewise, I was hoping that you would export a stub for `server` entries. For now this would be a loader that just exports the original file for non-`*.server.*` files. But for `*.server.*` files, you would generate a component that performs the fetch, e.g. probably by reading `renderServerComponent` from context, which would be provided by `server/render.tsx`.